<?xml version="1.0" ?>

<launch>

    <include file="$(find ur10_moveit_config)/launch/ur10_moveit_planning_execution.launch">
        <arg name="sim" value="true"/> 
        <arg name="limited" value="true"/>
    </include>

    <node name="main" pkg="ur10" type="ur10_main_node" output="screen"/>

    <include file="$(find ur10_moveit_config)/launch/moveit_rviz.launch">
        <arg name="config" value="true"/>
    </include>

</launch>
